gooddata-pandas/docs/conf.py

```diff
@@ -63,5 +63,11 @@
 
 source_suffix = {
     ".rst": "restructuredtext",
-    ".md": "markdown",
 }
+
+# auto-generate header anchors for 3 levels in Markdown files
+myst_heading_anchors = 3
```

Comment:
Why do we need it, when myst is not used?

Reply:
Removed